public/js/admin.js

```diff
@@ -744,7 +749,7 @@ var deleteQuestion = function(qid, questionNumber) {
                 if (data['status'] === 401) {
                     window.location.href = '/';
                 } else {
-                    failSnackbar('Coud not remove question ' + questionNumber + ' from the database');
+                    failSnackbar('Coud not remove question from the database');
```

Comment:
change this to the error code

Reply:
this is the functionality for the delete question which is not done yet, and so once this is implemented, error handling can occure